### PR TITLE
Corrections through 2019-08-10 + documentation improvements

### PR DESCRIPTION
**Pull request by mjpost**
Corrections to PDFs and some metadata corrections. Also updating the docs a bit.

### CODE REVIEW

**Comment by mjpost** [comment]
@mbollmann do you have any issues if I convert the FAQ to a single page, instead of automatically assembled sub-pages? It'd be slightly easier to manage entries.

**Comment by mjpost** [comment]
@mbollmann wrote:

> > Okay, I moved the links to the header in #538. It collapses correctly on mobile according to my testing (you can check it out [here](https://aclweb.org/anthology/index13.html)).
> 
> Hmm, but it no longer respects the width of the content on desktop, it seems:
> 
> image

The latest push fixes this.

> The collapsing is cool! The search button being wrapped onto the next row needs fixing though, and it'd be great if the search box appeared on top in the dropdown, if possible. Bootstrap should have classes to achieve that, I believe...

I don't have the CSS skills to fix these. What do you think about considering this a step forward and fixing it in a later commit?

**Comment by mjpost** [comment]
In an attempt to get this PR accepted, I moved the UI links to another branch. This commit now has only (a) documentation improvements and (b) corrections submitted to the Anthology through August 10.

File: hugo/content/faq/citing.md [hunk 1]
```diff
@@ -1,10 +0,0 @@
----
-Title: How do I cite publications in the ACL Anthology?
-weight: 1
----
-
-If available, you can you use Digital Object Identifiers (DOIs).
-The canonical URL for papers is given by appending the ACL ID to the Anthology URL, e.g., https://www.aclweb.org/anthology/E91-1001
-Both the DOIs and the persistent URLs embed the 8-character ACL Anthology Identifier.
-
-You can use variations on the canonical URL to access [the paper page](https://www.aclweb.org/anthology/E91-1001) (by appending a `/`) or [the BibTeX file](https://www.aclweb.org/anthology/E91-1001.bib) (by appending `.bib`).
```
Comment (mbollmann) [inline hugo/content/faq/citing.md]: Any particular reason this was deleted?

Comment (mjpost) [reply hugo/content/faq/citing.md]: I think I got thrown off by the word “cite”—you cite papers by using the BibTeX or other formats we export! But now I think the question is probably one about the canonical URLs. I’ll rewrite this in terms that make this more clear and add it back in.

Comment (mjpost) [reply hugo/content/faq/citing.md]: Okay, I rewrote this page.

File: bin/extract_attachments_from_csv.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
```
Comment (akoehn) [inline bin/extract_attachments_from_csv.py]: Please add a license header :-)

File: hugo/content/faq/volunteer.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ Title: How can I help improve the ACL Anthology?
 weight: 4
 ---
 
-The best way is to browse our [Github issues](https://twitter.com/TheGarlandSitIn/status/1123628614342385666) page, and see what needs to be done that might overlap with your interests or skills.
+We welcome your contributions!
+In fact, since the Anthology is run entirely on volunteer energy, we need them.
+The best way to contribute is to browse our [Github issues](https://twitter.com/TheGarlandSitIn/status/1123628614342385666) page, and see what needs to be done that might overlap with your interests or skills.
```
Comment (akoehn) [inline hugo/content/faq/volunteer.md]: JHU sit-in twitter message linked instead of the github issues?!  Interesting that nobody caught that before.

Comment (mjpost) [reply hugo/content/faq/volunteer.md]: Oh boy.

**Pull request merged**